iree/compiler/Translation/LLVM/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_subdirectory(test)
+
+iree_cc_library(
+  NAME
+    IREEToLLVM
+  HDRS
+    "LLVMLoweringPass.h"
+  SRCS
+    "LLVMLoweringPass.cpp"
+  DEPS
+    iree::compiler::Translation::XLAToLinalg::XLAToLinalg
```

Comment:
It should be sufficient to just depend on `iree::compiler::Translation::XLAToLinalg`.

Reply:
Don!